### PR TITLE
[HDInsight]Fix Error Response Structure

### PR DESCRIPTION
**Pull request by aim-for-better**
Fix Response Structure of ListHosts API

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/virtualMachines.json [hunk 1]
```diff
@@ -142,14 +142,9 @@
     },
     "HostInfoListResult": {
       "description": "Result of the request to list cluster hosts",
-      "properties": {
-        "value": {
```
Comment (pilor) [inline specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/virtualMachines.json:146]: was this a swagger mistake or is the API actually changing?

**Review by pilor — CHANGES_REQUESTED** [review]
:clock1:

File: specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/virtualMachines.json (hunk 1 above)
Comment (aim-for-better) [reply specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/virtualMachines.json:146]: > was this a swagger mistake or is the API actually changing?

It was swagger mistake. Thanks.

**Review by pilor — APPROVED** [review]

**Review by fengzhou-msft — APPROVED** [review]

**Comment by fengzhou-msft** [comment]
@akning-ms please help merge this PR. It fixes a wrong swagger response structure.

**Comment by aim-for-better** [comment]
> @akning-ms please help merge this PR. It fixes a wrong swagger response structure.

Thanks @fengzhou-msft  and @akning-ms

**Pull request merged**